test/mixture_laws.jl

```diff
@@ -85,3 +90,38 @@ end
     @test eltype(reddening) <: Gain
     @test ustrip.(reddening) ≈ collect(y) rtol = tolerance
 end
+
+
+@testset "G03_LMCAve" begin
+
+    # Test output
+    model = G03_LMCAve()
+    x, y, tolerance = model.obsdata_x, model.obsdata_axav, model.obsdata_tolerance
+    w = 1e4 ./ x
+    @test model.(collect(w)) ≈ collect(y) rtol = tolerance
+
+    # Test out of bounds
+    model = G03_LMCAve(Rv=2.74)
+    bad_waves = [100, 4e4]
+    @test model.(bad_waves) == zeros(length(bad_waves))
+    @test_throws ErrorException g03_invum(aa_to_invum(bad_waves[1]), 3.1)
+    @test_throws ErrorException g03_invum(aa_to_invum(bad_waves[2]), 3.1)
```

Comment:
Are you intending to test `g03lmc_invum` here? Currently these tests fail because `g03_invum` throws a `DomainError` while your  `g03lmc_invum` throws a generic `ErrorException`. I prefer the `DomainError` here.
```suggestion
    @test_throws DomainError g03lmc_invum(aa_to_invum(bad_waves[1]), 3.1)
    @test_throws DomainError g03lmc_invum(aa_to_invum(bad_waves[2]), 3.1)
```

Reply:
Actually, I just cloned what I found (when I forked the package) for the  g03smc analogous function, You proposals are of course fine for me.